charts/osm/values.yaml

```diff
@@ -19,6 +19,7 @@ OpenServiceMesh:
     protocol: http
     token:
     role: openservicemesh
+  connectVault: true
```

Comment:
A comment on what this does will be helpful.

Reply:
Even better than a comment, an entry in values.schema.json with a description.